src/Consumer.php

```diff
@@ -79,6 +79,9 @@ public function daemon($connectionName, $queue, WorkerOptions $options)
 
         [$startTime, $jobsProcessed] = [hrtime(true) / 1e9, 0];
 
+        $phpAmqpLibConnection = $this->container['config']["queue.connections.$connectionName.connection"];
```

Comment:
I think we'd want to only set the name if this config value is set.

I'm not sure everyone would like to see `\PhpAmqpLib\Connection\AMQPLazyConnection` start appearing by default as the connection name in the UI.